public/planner/js/engine/building.js

```diff
@@ -120,10 +128,35 @@ Building.prototype.moveHighlight = function moveHighlight(noFill) {
 };
 
 Building.prototype.mouseover = function mouseover(e) {
+
+    if (this.placed) {
+        const x = +this.sprite.attr('x');
```

Comment:
Would it make more sense for the function to be in the Building class because there is a tooltip per building? Or is the best option the Board class?

Reply:
As reported above with a general comment, the tooltip is currently missing from all placeable objects. 
I assume this was not intended and it should should tooltip for everything and then it makes sense to have it in Board class because it is a general tooltip function not directly tied to buildings.